lib/presentation/redux/states/my_space_state.dart

```diff
@@ -39,30 +41,55 @@ import 'package:linshare_flutter_app/presentation/redux/states/linshare_state.da
 
 @immutable
 class MySpaceState extends LinShareState {
-  final List<Document> documentList;
+  final Map<Document, bool> documentList;
+  final bool isSelectingDocuments;
```

Comment:
How about strong typing the state with an enum?
```dart
enum SelectMode { ACTIVE, INACTIVE }
```

Reply:
Sure why not I'll give it a try

Reply:
Done it, it looks cleaner thx